HW03/Ivan_Papushoi/mentor.rb

```diff
@@ -0,0 +1,40 @@
+class Mentor
+  attr_accessor :name, :students
+
+  def initialize(name, students)
+    @name = name
+    @students = students
+  end
+
+  def create_task(task_id)
+    Task.new(task_id)
+    "Mentor #{name} create task ##{task_id} \n" + notify_students(task_id)
+  end
+
+  def notify_students(task_id)
+    message = " was notified about task #{task_id} \n"
+    notify = ''
+    @students.each { |student| notify += 'Student ' + student.name + message }
+    notify
```

Comment:
Could you implement this logic on the `Student` class level?
```suggestion
    students.each { |student| student.notify(task_id) }
```

Reply:
I implemented, looks better